CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing Guidelines
+
+Welcome to the [warnet-gi/WarnetBot](https://github.com/warnet-gi/WarnetBot) 👋\
+Thanks for helping out the community!
+
+Before creating your pull request, make sure you already **read the whole guidelines**. If you feel that this guidelines is not clear enough or have any doubts, feel free to raise it in the [issue](https://github.com/warnet-gi/WarnetBot/issues/new) or by joining our [Discord channel](https://discord.gg/warnet-gi).
+
+## Project Structures
+
+- All bot contents are stored in `/bot`.
+  - the `/bot/assets` stores static assets.
+  - the `/bot/cogs` stores the bot logic.
+  - the `/bot/data` stores script to initialize PostgresSQL database.
+
+## Start Running in Local
+
+**Prerequisites**:
+
+- Python v3.10 (exact version)
+- PostgresSQL
+
+**Steps**:
+
+1. Fork or clone this repository.
+2. Go to Discord developers section and create a **[New Application](https://discord.com/developers/applications)**.
+3. After creating a new app, you will redirected to the app dashboard. Go to **Bot** menu in the sidebar.
+4. Find the **Token** section, then create a token.
+5. Copy the generated token and copy paste it to `BOT_TOKEN` in the `.env` file.
+6. You can use our provided `.env.example` in the root directory, just rename it to `.env`.
+7. Create a python virtual environment by running `python -m venv env` in the terminal to our project directory (there will be a `env/` folder in the root directory).
+8. Execute this command to activate virtual environment:
+
+- **Linux**: `source env/Scripts/Activate`
+- **macOS**: `source env/bin/activate` (macOS)
+- **Windows**: `env/Script/Activate.bat` (Windows).
+
+9. Install `poetry` by running `pip install poetry`.
```

Comment:
I think we need to add more context like "In the current active virtual env, install poetry ..."